pyproject.toml

```diff
@@ -77,6 +78,7 @@ cryptography = ["cryptography"]
 full = ["httpx", "brotli", "cryptography", "picologging", "structlog"]
 picologging = ["picologging"]
 structlog = ["structlog"]
+redis_cache_backend = ["redis"]
```

Comment:
```suggestion
redis = ["redis"]
```

Reply:
Done in: https://github.com/starlite-api/starlite/pull/564/commits/6a844b921676d88c613e39cb83796f09b2333a46